src/js/taskList.js

```diff
@@ -1,9 +1,17 @@
+/* eslint-disable */
+/* Main Class to set local storage */
+
 export default class {
-  static setList(tasks) {
+  setList(tasks) {
     return localStorage.setItem('tasks', JSON.stringify(tasks));
   }
 
-  static getList() {
+  getList() {
     return localStorage.getItem('tasks') ? JSON.parse(localStorage.getItem('tasks')) : [];
   }
+
+  reOrderTaskIndex(tasks) {
+    tasks.forEach(({index}, i) => index = i + 1);
+    return this.setList(tasks);
+  }
```

Comment:
- [ ] This function doesn't reorder the tasks correctly after a task or group of tasks have been removed. Sometimes, you end up with 2 or more todo tasks with the same index as shown below 👇 
![image](https://user-images.githubusercontent.com/46839548/171636622-3c777fdd-8d44-4333-b84a-10ceae8ffa8b.png)

Reply:
@codepantha Thanks for the feedback, I just missed this part while refactoring the code. My bad. I will keep more cautious in the future before requesting a review.